examples/ng9/src/app/assets-image/assets-image.component.cy-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { initEnv, mount } from 'cypress-angular-unit-test';
+import { AssetsImageComponent } from './assets-image.component';
+
+describe('AssetsImageComponent', () => {
+  it('should create', () => {
+    initEnv(AssetsImageComponent);
+    mount(AssetsImageComponent);
+    cy.get('img')
+      .should('be.visible')
+      .and(($img) => {
+        expect($img[0].naturalWidth).to.be.greaterThan(0);
```

Comment:
nice, the image is loaded

Reply:
Nope, the image path is not good.
The `staticResourceLoader` is not called...

Reply:
where `/__root` is routing?